src/components/Scrollable/tests/Scrollable.test.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import Scrollable from '../Scrollable';
+
```

Comment:
Add root level `describe('<Scrollable />', () => {` describe and have all tests inside it.